paper.md

```diff
@@ -357,7 +357,7 @@ By setting up CI, users can continuously validate changes made to their workflow
 A summary of the training duration and accuracy obtained by running the workflow in three different computing environment is shown in Figure @Fig:casestudies.
 It can be seen from the graph how the portability of Popper workflows drastically reduces software development and debugging time by enabling developers and researchers to quickly iterate and test in different computing environments.
 
-![Popper configuration files for different computing environments](./figures/config.jpg){#fig:conffiles}
+![Popper configuration files for different computing environments](./figures/config.png){#fig:conffiles}
```

Comment:
let's add this as plain text instead, so we can avoid having to deal with issues of low resolution images, coloring, etc.

Reply:
Yeah. done that.